pages/emoticons_page.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: UTF-8 -*-
+
+from .base_page import Page
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+import os
+
+
+class EmoticonsPage(Page):
+
+    url = '/emoticons'
+
+    def go_to_constructor(self):
```

Comment:
This not mistake at all, but, I don't understand why you named method so ununderstandable, I gess what about this method from id of element(filepicker) that you search. We should to do our tests/steps/methods more useble for others teammates, if it possible.

Reply:
U r right. Better to set name of "filepicker" not "constructor. It will be more informative. Done